libraries/apollo-gradle-plugin/build.gradle.kts

```diff
@@ -128,4 +128,7 @@ tasks.withType<Test> {
 
   inputs.dir("src/test/files")
   inputs.dir("testProjects")
+
+  maxParallelForks = (Runtime.getRuntime().availableProcessors() / 2).takeIf { it > 0 } ?: 1
```

Comment:
I believe we can use `Runtime.getRuntime().availableProcessors()` here, which means full cores, there are 2 cores in GHA runners, use `(Runtime.getRuntime().availableProcessors() / 2).takeIf { it > 0 } ?: 1` (result is 1) here seems to be less improvement on CI.